### ISSUE TITLE
Desktop Tower Defense (USA) seems to softlock on the profile selection screen

### ISSUE THREAD

**Issue by Tiberius1437** [Arisotura/melonDS#871]
Maybe I'm not understanding something but I can't interact with it at all. It's the same with Desmume. 

![Capture](https://user-images.githubusercontent.com/68873449/102734957-4d3d8e80-4395-11eb-8b7c-00bee0580f96.PNG)

**Comment by Arisotura** [comment]
I can interact with it just fine. you click one of the Empty buttons and then click Create.

**Comment by RSDuck** [comment]
did you try it with JIT enabled?

**Comment by Arisotura** [comment]
nope, plain ol' interpreter. are there different effects with various JIT modes? if so, I'm leaving that issue to you :P

otherwise, we can probably close this.

**Comment by Tiberius1437** [comment]
That's weird. I asked someone else about it earlier today and they had the same problem. It's not a JIT issue, I turned that off. I also set the renderer to software. 

![Capture](https://user-images.githubusercontent.com/68873449/102791242-d0450000-43fa-11eb-83d9-02233a26cf94.PNG)

**Comment by RSDuck** [comment]
yeah, the title screen is just weird. You need to put your cursor next to the button and then hold while moving it into the button.

**Comment by Arisotura** [comment]
I observed I had to actually click when using the touchpad (not just tap), but thought that was just weird touchpad shenanigans.

I'll look more into it at home. I recall DeSmuME having some code to simulate touchscreen coord jitter, but if even that doesn't help, well...

**Comment by RSDuck** [comment]
Somehow it works on real hw. Both swiping into a button, but also directly pressing on one button.

**Comment by iamthelaw2041** [comment]
Something is definitely fucked with this. I tested it on a 3DS and it worked perfectly. In MelonDS clicking does nothing. I have to randomly mash all around the screen to get inputs to register. 

![Capture](https://user-images.githubusercontent.com/94656803/145669267-33c8ef11-14b7-4c06-9720-3e0bbc9df1c0.PNG)

**Comment by RSDuck** [comment]
it's really pretty simple, you just need to click and move a bit inside the button.
